actors/builtin/miner/policy.go

```diff
@@ -99,6 +99,10 @@ const WorkerKeyChangeDelay = ChainFinality
 // and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
 const MaxSectorExpirationExtension = builtin.EpochsInYear
 
+// MaxPoStChainCommitAge is the maximum distance back that a PoSt can commit to
+// a chain before being invalid
+const MaxPoStChainCommitAge = ChainFinality
```

Comment:
it is better if this is smaller than finality - we can give you a better number for this.

Reply:
We should set this to 5.